### PR TITLE
$folder['permissions'] is always null

### PR DESCRIPTION
**Pull request by CarlSchwan**
Signed-off-by: Carl Schwan <carl@carlschwan.eu>

### CODE REVIEW

File: lib/Command/ACL.php [hunk 1]
```diff
@@ -150,7 +151,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 			$mount = $this->mountProvider->getMount(
 				$folder['id'],
 				'/dummy/files/' . $folder['mount_point'],
-				$folder['permissions'],
+				Constants::PERMISSION_ALL,
 				$folder['quota'],
 				$folder['rootCacheEntry'],
```
Comment (come-nc) [inline lib/Command/ACL.php:156]: From what I’ve seen rootCacheEntry is also not present in $folder array

**Review by come-nc — CHANGES_REQUESTED** [review]

File: lib/Command/ACL.php (hunk 1 above)
Comment (CarlSchwan) [reply lib/Command/ACL.php]: Good catch, fixed!

**Review by come-nc — APPROVED** [review]

**Pull request merged**